### ISSUE TITLE
 _babylonWalk(...).traverseAll() is not a function ? Really ?

### ISSUE THREAD

**Issue by Pi-Bouf** [parcel-bundler/parcel#6399]
# 🐛 bug report
Impossible to build a simple App.vue with a typescript createApp(...) on Parcel V2

## 🎛 Configuration (.babelrc, package.json, cli command)

This is all my package.json:
```js
  "dependencies": {
    "@babel/preset-typescript": "^7.13.0",
    "@babel/types": "^7.14.4",
    "@parcel/babylon-walk": "^2.0.0-nightly.1765",
    "@yaireo/console-colors": "^1.1.0",
    "bytebuffer": "^5.0.1",
    "core-js": "3",
    "gsap": "^3.6.1",
    "pixi.js": "^6.0.4",
    "ts-bus": "^2.3.1",
    "v-tooltip": "^2.1.3",
    "vue": "^3.0.11",
    "vuex": "^4.0.1"
  },
  "devDependencies": {
    "@babel/core": "^7.14.3",
    "@babel/plugin-transform-runtime": "^7.14.3",
    "@babel/preset-env": "^7.14.2",
    "@parcel/transformer-vue": "2.0.0-beta.3.1",
    "@types/bytebuffer": "^5.0.42",
    "@typescript-eslint/eslint-plugin": "^4.24.0",
    "@vue/component-compiler-utils": "^3.2.0",
    "eslint": "^7.27.0",
    "eslint-config-airbnb-typescript": "^12.3.1",
    "eslint-plugin-import": "^2.23.3",
    "javascript-obfuscator": "^2.12.0",
    "parcel": "^2.0.0-beta.3.1",
    "sass": "^1.34.0",
    "typescript": "^4.2.4"
  },
  "alias": {
    "Mixins": "./assets/sass/_mixins.scss"
  }
```

## 🤔 Expected Behavior
A fully builded vue component ? :D

## 😯 Current Behavior
Only a beautiful error on the BabylonWalk:
```
@parcel/transformer-babel: (0 , _babylonWalk(...).traverseAll) is not a function
TypeError: (0 , _babylonWalk(...).traverseAll) is not a function
    at remapAstLocations (node_modules\@parcel\babel-ast-utils\lib\index.js:105:34)
    at babel7 (node_modules\@parcel\transformer-babel\lib\babel7.js:117:48)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async Object.transform (node_modules\@parcel\transformer-babel\lib\BabelTransformer.js:124:11)
    at async Transformation.runTransformer (node_modules\@parcel\core\lib\Transformation.js:821:5)
    at async Transformation.runPipeline (node_modules\@parcel\core\lib\Transformation.js:577:36)
    at async Transformation.runPipelines (node_modules\@parcel\core\lib\Transformation.js:426:40)
    at async Transformation.runPipelines (node_modules\@parcel\core\lib\Transformation.js:462:34)
    at async Transformation.run (node_modules\@parcel\core\lib\Transformation.js:326:19)
```

## 🔦 Context

Build a simple App.vue :'(

## 💻 Code Sample

This is my App.vue:
```js
<template>
  <div id="hotel-base">
  </div>
</template>

<script>
import {defineComponent} from 'vue';

export default defineComponent({
  name: 'App'
})
</script>

<style lang="scss">
#hotel-base {
  background-color: #000000;
  position: fixed;
  left: 0;
  right: 0;
  bottom: 0;
  top: 0;
}
</style>

```

And my app.ts
```ts
const app = createApp({
                template: '<app></app>',
                components: {
                    App,
                },
                mounted() {
                    resolve();
                },
            });

            app.use(store);
            app.use(VTooltip);
            app.component('v-popover', VPopover);

            app.mount(Configuration.global.targetInterface);
```

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 2.0.0-beta.3.1
| Node             | 14.15.4
| npm/Yarn         | 1.22.10 (yarn)
| Operating System | Win 10 (latest insider)

**Comment by puka-tchou** [comment]
I'm having the same error when trying to build Boostrap, I hope there's a solution to this.

**Comment by mischnic** [comment]
Can you try the latest nightly? This should be fixed by https://github.com/parcel-bundler/parcel/pull/6388.

**Comment by Pi-Bouf** [comment]
> I'm having the same error when trying to build Boostrap, I hope there's a solution to this.

Yes !! I just find something: build with --no-source-maps
And now, it works...  but you need to do it without the source maps :/

**Comment by Pi-Bouf** [comment]
> Can you try the latest nightly? This should be fixed by #6388.

Ok, I tried
`yarn add @parcel/babylon-walk@nightly``, it works perfectly ! THANK YOU <3
